Dokan/DokanTests/Scenes/Registratio9nTest/RegisterViewControllerTests.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  RegisterViewControllerTests.swift
+//  DokanTests
+//
+//  Created by heba isaa on 05/07/2022.
+//
+
+import XCTest
+@testable import Dokan
+class RegisterViewControllerTests: XCTestCase {
```

Comment:
add new line after the importing 
```suggestion

class RegisterViewControllerTests: XCTestCase {
```